sqlacodegen/codegen.py

```diff
@@ -158,8 +158,22 @@ def add_imports(self, collector):
             if len(index.columns) > 1:
                 collector.add_import(index)
 
+    @staticmethod
+    def _convert_to_valid_identifier(name):
```

Comment:
Why did you move the method here? It doesn't seem to be used outside of ModelTable even in your version.

Reply:
Previously **_convert_to_valid_identifier** was only used in ModelClass and it was a static method of this class. Now **_convert_to_valid_identifier** is also used in ModelTable so it is moved to the base class.